src/library_html5.js

```diff
@@ -2359,7 +2359,7 @@ var LibraryHTML5 = {
 
   emscripten_set_batterychargingchange_callback_on_thread__proxy: 'sync',
   emscripten_set_batterychargingchange_callback_on_thread__sig: 'iii',
-  emscripten_set_batterychargingchange_callback_on_thread__deps: ['$registerBatteryEventCallback', '$battery'],
+  emscripten_set_batterychargingchange_callback_on_thread__deps: ['$registerBatteryEventCallback', '$battery', 'malloc'],
```

Comment:
This and the one after it don't call malloc directly, so why do they need it in their deps?

Reply:
I've got a followup that handling indirect deps.  I will remove this as part of that change if thats OK?